Formula/terraform.rb

```diff
@@ -63,6 +57,8 @@ def install
     end
 
     cd terrapath do
+      system "godep", "restore"
```

Comment:
Godep seems to be failing to resolve the standard library references. I tried running `godep restore` with the debug flag and it seems that godep isn't searching `GOROOT/src` unless `GOROOT` is explicitly specified as an environment variable. I was able to fix this by explicitly setting a GOROOT, but it feels kind of dirty:

```
...
def install
     ENV["GOPATH"] = buildpath
     ENV["GOROOT"] = `go env GOROOT`.chomp
...
```
```